packages/glibc.rb

```diff
@@ -76,6 +85,17 @@ def self.patch
         end
       end
     when '2.32'
+      FileUtils.mkdir 'fedora'
+      # Patch to enable build-local-archive
+      system 'curl -Ls https://src.fedoraproject.org/rpms/glibc/raw/f30/f/glibc-fedora-locarchive.patch | \
+      hashpipe sha256 0acccf57d8c6e7de82871c61ccb845f7a1ae13ae1fbc65995d347de8367c7bb1 | \
+      patch -Np1 --binary'
+      system 'curl -Ls https://src.fedoraproject.org/rpms/glibc/raw/f30/f/build-locale-archive.c | \
+      hashpipe sha256 6834e8b8f2a987bf8adfd265c0e01665f102c7115db94b99ec36376b68e9d3fa > fedora/build-locale-archive.c'
+      system "sed -i 's,/lib/locale,/lib#{CREW_LIB_SUFFIX}/locale,g' fedora/build-locale-archive.c"
+      system "sed -i 's,/usr/sbin/tzdata-update,/bin/true,g' fedora/build-locale-archive.c"
+      system "sed -i 's,verbose,locale_verbose,g' fedora/build-locale-archive.c"
+      system "sed -i 's,be_quiet,locale_be_quiet,g' fedora/build-locale-archive.c"
```

Comment:
I feel like we're making this way too complicated.  Why are we using the fedora sources?

Reply:
See one rationale here, via another project using the same tool: https://github.com/Mrm3/image-build/blob/4a878611f813f1a859e48747ecf68897e7dd9015/diskimage_builder/elements/yum-minimal/pre-install.d/03-yum-cleanup#L38

In short, using the Fedora tool lets us avoid this:

![image](https://user-images.githubusercontent.com/1096701/126001140-9806bc21-bc85-49ed-a006-342339c5c117.png)

Also, we get the benefit of users being able to modify the postinstall locally to add any other locales they might need with a quick reinstall of glibc.